AsyncDisplayKit/ASCellNode.h

```diff
@@ -74,6 +74,7 @@ typedef NS_ENUM(NSUInteger, ASCellNodeVisibilityEvent) {
  */
 @property (nonatomic, assign) BOOL neverShowPlaceholders;
 
+//TODO shouldn't this be a generic kind (e.g includes ASDataControllerRowNodeKind)?
```

Comment:
I guess we have a common ground that this property should be called `kind` and exposed as `nil` for row kind?

On the other hand, can `ASDataControllerRowNodeKind` possibly be exposed/public?